src/portable/credentials.rs

```diff
@@ -23,17 +36,24 @@ pub fn show_credentials(options: &Options, c: &ShowCredentials) -> anyhow::Resul
         if let Some(database) = creds.database {
             url = url.join(&database)?;
         }
-        match creds.tls_security {
-            TlsSecurity::Strict => {
-                url.set_query(Some(&format!("tls_security=strict")));
-            }
-            TlsSecurity::Insecure => {
-                url.set_query(Some(&format!("tls_security=insecure")));
-            }
-            TlsSecurity::NoHostVerification => {
-                url.set_query(Some(&format!("tls_security=no_host_verification")));
-            }
-            _ => {}
+        let query = [
+            (
+                "tls_security",
+                if creds.tls_security == TlsSecurity::Default {
+                    None
+                } else {
+                    Some(serde_json::to_value(&creds.tls_security).unwrap().as_str().unwrap().into())
+                },
+            ),
+            ("secret_key", secret_key),
+        ]
+        .iter()
+        .map(|(k, v)| v.as_deref().map(|v| format!("{}={}", k, v)))
+        .flatten()
+        .collect::<Vec<_>>();
```

Comment:
I use same indentation level for dot-something only if that is the only thing in that code block and still for short blocks. Otherwise it hinders readability.